testsuite/features/step_definitions/command_steps.rb

```diff
@@ -420,7 +420,11 @@
       sleep checking_rate
     end
   rescue StandardError => e
-    log e.message # It might be that the MU repository is wrong, but we want to continue in any case
+    log e.message
+    # It might be that the MU repository is wrong, but we want to continue in any case
```

Comment:
move the comment after the "unless" for clarity

Reply:
Done